### PR TITLE
Added react helmet for handling SEO

### PR DESCRIPTION
**Pull request by AsifOni**
Added react helmet to support SEO stuff at page level

### CODE REVIEW

File: src/pages/Quote.tsx [hunk 1]
```diff
@@ -129,6 +126,10 @@ const Quote = () => {
 
   return (
     <div className="flex flex-col gap-x-6 px-4 py-6 lg:grid lg:grid-cols-5 lg:px-9 lg:py-8">
+      <Helmet>
```
Comment (alaminShaheen) [inline src/pages/Quote.tsx]: Bhai helmet also has a title prop. You can set title through the title prop.

**Review by farzanashampa7 — APPROVED** [review]
Looks good

File: src/pages/Quote.tsx (hunk 1 above)
Comment (AsifOni) [reply src/pages/Quote.tsx]: @alaminShaheen - I couldn't find any mention about the prop in the documentation.

https://github.com/staylor/react-helmet-async#readme

Comment (alaminShaheen) [reply src/pages/Quote.tsx]: It's not properly documented bhai, but its there. Typescript should show the props available.

Comment (AsifOni) [reply src/pages/Quote.tsx]: @alaminShaheen  - You were right. Found prop definitions. Adjusted code

image

**Pull request merged**